mmv1/products/apigee/EnvironmentAddonsConfig.yaml

```
@@ -0,0 +1,66 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: 'EnvironmentAddonsConfig'
+api_resource_type_kind: AddonsConfig
+description: Enable/Disable add-ons for an Apigee environment.
+references:
+  guides:
+    'Enable Analytics Add-On': 'https://cloud.google.com/apigee/docs/api-platform/reference/manage-analytics-add-on'
+    'Enable ApiSecurity Add-On': 'https://cloud.google.com/apigee/docs/api-platform/reference/manage-security-add-on'
+  api: 'https://cloud.google.com/apigee/docs/reference/apis/apigee/rest/v1/organizations.environments.addonsConfig/setAddonEnablement'
+docs:
+base_url: '{{env_id}}/addonsConfig'
+self_link: '{{env_id}}/addonsConfig'
+create_url: '{{env_id}}/addonsConfig:setAddonEnablement'
+update_url: '{{env_id}}/addonsConfig:setAddonEnablement'
+update_verb: 'POST'
+exclude_delete: true
+timeouts:
+  insert_minutes: 5
+  update_minutes: 5
+async:
+  actions: ['create', 'update']
```

(if delete support is added)

```suggestion
  actions: ['create', 'update', 'delete']
```

---

same as previous comment on supporting delete